README.md

```diff
@@ -86,81 +119,107 @@ Configure this action by either inlining these options in your workflow file, or
+        external-repo-token: 'ghp_123456789abcde'
+   ```
+
+   | Option                | Usage    | Possible values | 
+   |--------------------- | ----------- | ----------------------------- |
+   | `external-repo-token` | Specifies a token for fetching the configuration file. It is required if the file resides in a private external repository and for all GitHub Enterprise Server repositories. Create a token in [developer settings](https://github.com/settings/tokens). | Any token with `read` permissions to the repository hosting the config file. |
+3. Create the configuration file in the path you specified for `config-file`.
+4. In the configuration file, specify your chosen settings.
+   ```yaml
+   fail_on_severity: 'critical'
+   allow_licenses:
+     - 'GPL-3.0'
+     - 'BSD-3-Clause'
+     - 'MIT'
+   ```
+> [!NOTE]
+> For external configuration files, the option names use underscores instead of dashes.
+> Example: `fail_on_severity`
 
-### Considerations
+#### Further information
 
-- Checking for licenses is not supported on Enterprise Server as the API does not return license information.
-- The `allow-licenses` and `deny-licenses` options are mutually exclusive; an error will be raised if you provide both.
-- We don't have license information for all of your dependents. If we can't detect the license for a dependency **we will inform you, but the action won't fail**.
+- For more examples of how to use this action and its configuration options, see the [examples](docs/examples.md) page.
+- For general information about dependency review on GitHub, see "[About dependency review](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review)" in the GitHub Docs documentation.
 
-## Blocking pull requests
+## Using dependency review action to block a pull request from being merged
 
-The Dependency Review GitHub Action check will only block a pull request from being merged if the repository owner has required the check to pass before merging. For more information, see the [documentation on protected branches](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging).
+You can configure your repository to block a pull request from being merged if the pull request fails the dependency review action check. To do this, the repository owner must configure branch protection settings that require the check to pass before merging. For more information, see "[Require status checks before merging](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging)" in GitHub Docs documentation.   
 
 ## Outputs
 
+[TODO: Insert overview line here - what does output refer to? What / how it is used?]
```

Comment:
It would be good to add some sort of intro line here? But I wasn't sure what to put...